build

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
last bikeshed: we should use bash here, since we rely on bashisms.
However, use `/bin/bash` or `/usr/bin/env bash`?
Since speedcenter is Debian Linux, we can be pretty sure bash will be installed to `/bin/bash`.

Reply:
I always use "/usr/bin/env bash", since its more portable. But in this case it dosen't matter. We could of course vote on slack ;-)